exercises/concept/string-vs-str/.docs/instructions.md

```diff
@@ -1,23 +1,36 @@
-We need a builder for a CSV record per the standard defined in [RFC 4180](https://tools.ietf.org/html/rfc4180).
+Your company has been hired to generate CSV documents from user input.
 
-The builder needs to concatenate immutable string slices (`&str`) and when `build` is called, the builder will be consumed and will return the created CSV record (one line of a CSV file).
+While a formal specification exists, your company only needs to implement the following functionality:
+- create a new builder
+- add an individual record
+- serialize records with basic escaping
+
+## 1. Create a new builder
+
+You'll need to implement the `new` method to create a new builder.
 
 ```rust
 let mut builder = CSVRecordBuilder::new();
+```
+
+## 2. Add individual records
 
+You'll need to implement the `add` method to append records to your builder.
+`add` takes immutable string slices, `&str`, as the argument.
+
+```rust
 builder.add("ant");
 builder.add("ba\"t");
 builder.add("cat");
-
-let list = builder.build();
-//Note that from now on we can do nothing now with builder.
-assert_eq!(r#"ant,"ba""t",cat"#, &list);
 ```
 
-## Edge cases
```

Comment:
It would be nice to retain this section's information somewhere, so students don't actually have to chase down the necessary rules in the formal specification. Maybe `about.md`?

Reply:
Great idea. I will preserve it there!